src/amber/dsl/router.cr

```diff
@@ -28,6 +28,13 @@ module Amber::DSL
     {% for verb in RESOURCES %}
       macro {{verb.id}}(*args)
         route {{verb}}, \{{*args}}
+        {% if verb == :get %}
+          route :head, \{{*args}}
```

Comment:
I think we should align this line

BTW, macros aren't checked by crystal tool format, they keep untouched